src/test/java/edu/cornell/kfs/module/purap/batch/service/impl/JaggaerGenerateContractPartyCsvServiceImplTest.java

```diff
@@ -187,5 +212,151 @@ static Stream<Arguments> jaggaerContractPartyUploadDtoToStringParameters() {
                 Arguments.of(buildVendor1Dto(),  "taxIdentificationNumber=,")
                 );
     }
+    
+    @ParameterizedTest
+    @MethodSource("builderVendorCSVRowArrayParameters")
+    void testBuilderVendorCSVRowArray(JaggaerContractPartyUploadDto vendorDto, String[] expectedCsvData) {
+
+        String[] actualCsvData = jaggaerGenerateContractPartyCsvServiceImpl.builderVendorCSVRowArray(vendorDto);
+        assertEquals(16, expectedCsvData.length);
+        assertEquals(16, actualCsvData.length);
+        validateCsvRowArray(expectedCsvData, actualCsvData);
+    }
+
+    public void validateCsvRowArray(String[] expectedCsvData, String[] actualCsvData) {
+        for (int i = 0; i < expectedCsvData.length; i++) {
+            String actualDatum = actualCsvData[i];
+            String expectedDataum = expectedCsvData[i];
+            assertEquals(expectedDataum, actualDatum);
+        }
+    }
+
+    static Stream<Arguments> builderVendorCSVRowArrayParameters() {
```

Comment:
There are various test methods and arg-building methods that say "builder" instead of "build". Is that the way you meant to name them?

Reply:
yes, switching to buildXXParameters